CMakeLists.txt

```diff
@@ -51,6 +51,20 @@ if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "armv7l")
   set(CMAKE_CXX_LINK_FLAGS "${CMAKE_CXX_LINK_FLAGS} -latomic")
 endif()
 
+if (${CMAKE_SYSTEM_NAME} MATCHES "Windows")
```

Comment:
maybe we can move this code to cmake/windows-setup.cmake ?

https://github.com/fluent/fluent-bit/blob/master/cmake/windows-setup.cmake

Reply:
Sure, I'll do.

Reply:
Done by https://github.com/fluent/fluent-bit/pull/4531/commits/92a357bbb96a7f466e208a43dc289d3007a62f0d.